src/wallet/wallet.cpp

```diff
@@ -1420,6 +1420,37 @@ void CWallet::transactionAddedToMempool(const CTransactionRef& tx) {
     if (it != mapWallet.end()) {
         RefreshMempoolStatus(it->second, chain());
     }
+
+    const Txid& conflict_txid = tx->GetHash();
+
+    for (const CTxIn& tx_in : tx->vin) {
+        // If the wallet contains no transactions that spend the same prevout, continue
+        if (mapTxSpends.count(tx_in.prevout) == 0) continue;
```

Comment:
In commit "wallet: track mempool conflicts" (2c867784b2a719c9e701e19e389c3adfa7d7faae)

There's no reason for this line. If the count is 0, equal_range will return an empty range and the result will be the same.

Reply:
Fixed here and for `transactionRemovedFromMempool`.